python/lsst/ts/MTMount/mtmount_csc.py

```diff
@@ -503,7 +509,7 @@ async def send_commands(self, *commands, dolock=True):
         ----------
         commands : `List` [``Command``]
             Commands to send. The sequence_id attribute is set.
-        dolock : `bool` (optional)
+        dolock : `bool`, optional
```

Comment:
I notice that only the doc string has changed but wouldn't "do_lock" be a clearer name?

Reply:
That's fair. I changed it.